### DISCUSSION TITLE
关于自定义主题

### DISCUSSION THREAD

**Discussion by Vickers7406** [Meloong-Git/PCL#1817] [讨论]
自定义主题建议改成让玩家输入颜色代码如#314159或（110,119,120），现在的色调居然是按360°改的......

**Comment by Chiloven945** [comment]
Hex，RGB，HSL的支持
可以交一个issue

**Reply by iPodtouch6557** [reply]
但是大概率会被拒绝或者延期……

**Reply by Chiloven945** [reply]
> 但是大概率会被拒绝或者延期……

应该不是一件很困难的事情……

**Comment by WForst-Breeze** [comment]
在金数据时期此项内容因技术原因被拒绝制作。不过目前是否可以实现尚未知晓，您可以提交优化建议。

**Comment by Griseo-bh3rd** [comment]
#401 已拒绝此功能

**Reply by iPodtouch6557** [reply]
mark as answer……？

**Reply by Griseo-bh3rd** [reply]
> mark as answer……？

mark不了吧。。。这是讨论耶

**Reply by iPodtouch6557** [reply]
> > mark as answer……？
> 
> mark不了吧。。。这是讨论耶

这个人就是瞎交 category ……
